### ISSUE TITLE
[BUG] Player.json erases itsself

### ISSUE THREAD

**Issue by LinkTopia2** [GrowtopiaNoobs/GrowtopiaServer#1303]
I'm having an error, where once in a while, one or more json player files, delete all their data, this causes exe to crash. I'm thinking to add an auto save system, but I'm not sure. Any tips help, thanks!

**Comment by MarioBITs** [comment]
Just delete the 0kb data.

**Comment by LinkTopia2** [comment]
> Just delete the 0kb data.
> 
> **And add autosave worlds**

Yes, but the player.json deletes, not worlds. And when you try to log on the corrupted json file, it crashes exe

**Comment by MarioBITs** [comment]
> > Just delete the 0kb data.
> > **And add autosave worlds**
> 
> Yes, but the player.json deletes, not worlds. And when you try to log on the corrupted json file, it crashes exe

dont use leaked cpp then

**Comment by noobsteak** [comment]
Fix the Json saving

**Comment by xXAbood** [comment]
wtf man :[ u made rollback 4 u can do that too wtf how ?! 😳

**Comment by SSebia** [comment]
> > Just delete the 0kb data.
> > **And add autosave worlds**
> 
> Yes, but the player.json deletes, not worlds. And when you try to log on the corrupted json file, it crashes exe

You made mistake while modifying the contents of that .json, i suggest to revert the latest updates you have done. This is possible that you are getting value from json that does not exist and that causes it to corrupt.
